src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -441,13 +440,23 @@ private void updateProcessingVenueIfRequired(CaseDetails<SscsCaseData> caseDetai
         String venue = airLookupService.lookupAirVenueNameByPostCode(postCode, sscsCaseData.getAppeal().getBenefitType());
```

Comment:
maybe this can be named latestLookupVenue or something similar to make it clear it's also the new venue.
or maybe simply call this newVenue, and the one below newVenueDetails